aea/cli/generate.py

```diff
@@ -50,13 +51,21 @@ def generate(ctx: Context):
 
 def _generate_item(ctx: Context, item_type, specification_path):
     """Generate an item based on a specification and add it to the configuration file and agent."""
-    # # check protocol buffer compiler is installed
-    # res = shutil.which("protoc")
-    # if res is None:
-    #     print(
-    #         "Please install protocol buffer first! See the following link: https://developers.google.com/protocol-buffers/"
-    #     )
-    #     sys.exit(1)
+    # check protocol buffer compiler is installed
+    res = shutil.which("protoc")
+    if res is None:
+        print(
+            "Please install protocol buffer first! See the following link: https://developers.google.com/protocol-buffers/"
+        )
+        sys.exit(1)
+
+    # check black code formatter is installed
+    res = shutil.which("black")
```

Comment:
an idea: since below it is used with `-m black`, we should 
```python
try:
    import black
except ImportError:
    print(...)
```

Reply:
removed `-m` instead